cmd.go

```diff
@@ -281,8 +281,12 @@ func handleSync() error {
 }
 
 func handleRemove() error {
-	removeVCSPackage(cmdArgs.targets)
-	return show(passToPacman(cmdArgs))
+	var err = show(passToPacman(cmdArgs))
```

Comment:
Hi, thanks for taking the time to fix this issue, Everything seems fine I would just like a small style change to `err :=` instead of explicitly declaring a var

Reply:
Right, my bad on that one, thanks!